cli/src/config/revsets.toml

```diff
@@ -21,3 +21,5 @@ latest(
 'immutable_heads()' = 'trunk() | tags()'
 'immutable()' = '::(immutable_heads() | root())'
 'mutable()' = '~immutable()'
+
+'date(d)' = 'committer_date(d)'
```

Comment:
Is this alias really needed? If user wants to compare committer timestamp, I think he can just use `committer_date()`.

Reply:
The only reason I added this was, while discussing the feature in Discord, some people expected the function to be `date()`. I will remove the alias though. If enough people feel it's missing, it can always be added later.

Reply:
Removed.